src/Spectre.libGaussianMixtureModelling/Matrix.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Matrix.h
+ * Provides temporary implementation of matrix class that stores
+ * its contents contiguously in the memory.
+ *
+Copyright 2017 Michal Gallus
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#pragma once
+#include "DataType.h"
+
+namespace Spectre::libGaussianMixtureModelling
+{
+    /// <summary>
+    /// Struct serves as a temporary matrix representation.
+    /// It stores its data contigiuously in the mermoy.
+    /// </summary>
+    struct Matrix
```

Comment:
It's not necessary - right now we will migrate to `Dataset`, while we are looking for ready matrix library. It's just gathering strange code into one place, to replace it quickly.